### PR TITLE
fix(nvim-toggler): add lazy load on keys

### PR DESCRIPTION
**Pull request by reaink**
plugin nvim-toggler keys tip toggle cursorword

### CODE REVIEW

**Comment by owittek** [comment]
Please fix the PR title & commit message to follow the conventions

File: lua/astrocommunity/utility/nvim-toggler/init.lua [hunk 1]
```diff
@@ -1 +1,12 @@
...
+    event = { "User AstroFile", "InsertEnter" },
+    keys = {
+      {
+        "<leader>i",
+        desc = "Toggle CursorWord",
+      },
+    },
+    opts = {},
+  },
+
```
Comment (mehalter) [inline lua/astrocommunity/utility/nvim-toggler/init.lua:12]:
```suggestion
  }
```

**Review by mehalter — CHANGES_REQUESTED** [review]
Small syntax error and please format the code with `stylua` so that the tests pass. Thanks for the contribution!

File: lua/astrocommunity/utility/nvim-toggler/init.lua (hunk 1 above)
Comment (mehalter) [inline lua/astrocommunity/utility/nvim-toggler/init.lua:12]:
```suggestion
return {
  "nguyenvukhang/nvim-toggler",
  event = { "User AstroFile", "InsertEnter" },
  keys = {
    {
      "<leader>i",
      desc = "Toggle CursorWord",
    },
  },
  opts = {},
}
```

**Review by mehalter — APPROVED** [review]
LGTM!

**Pull request merged**